src/Apps/Order/Components/TransactionDetailsSummaryItem.tsx

```diff
@@ -220,8 +231,16 @@ export const TransactionDetailsSummaryItem: FC<TransactionDetailsSummaryItemProp
         data-test="buyerTotalDisplayAmount"
       />
       <Spacer y={2} />
+      {offer?.shippingTotalCents &&
+      order.requestedFulfillment?.__typename === "CommerceShipArta" &&
+      order.mode === "OFFER" &&
+      order.displayState !== "PROCESSING" ? (
+        <Text variant="sm" color="black60">
+          *Estimate Only. Price may vary once offer is finalized.
+        </Text>
+      ) : null}
```

Comment:
one useful syntax for this kind of cases where we want to render X if Y evaluates to true and don't render anything if it's false, is to treat JSX like a condition, like so: 
`{order.mode === "OFFER" && order.displayState !== "PROCESSING" && .... && your JSX}`

Reply:
Thanks! Refactoring made